README.md

```diff
@@ -162,10 +162,12 @@ For a general introduction to installing modules, see
   and [black](https://black.readthedocs.io) style checker
   locally,
   as these checks are required to pass by the online integration pipeline.
-* Some of the [examples](./examples) require [gridcorner](https://gitlab.aei.uni-hannover.de/GregAshton/gridcorner);
+* Some of the [examples](./examples) require [gridcorner](https://gitlab.aei.uni-hannover.de/GregAshton/gridcorner) 
```

Comment:
Actually this comment was already incorrect for `gridcorner`, as it's not only called from the examples directly, but from methods actually implemented inside the PyFstat library modules.
```suggestion
* Some optional plotting methods depend on two additional packages,
  and some of the [examples](./examples) require these to run:
 [gridcorner](https://gitlab.aei.uni-hannover.de/GregAshton/gridcorner) 
```